simulation_ws/src/cloudwatch_simulation/launch/bookstore_turtlebot_navigation.launch

```diff
@@ -32,8 +32,6 @@
 
   <!-- Send navigation route goals to the robot in random order -->
   <group if="$(arg follow_route)">
-    <node pkg="aws_robomaker_simulation_common" type="route_manager" name="route_manager" output="screen">
-      <rosparam file="$(find aws_robomaker_bookstore_world)/routes/route.yaml" command="load"/> 
-    </node>
+    <node pkg="aws_robomaker_simulation_common" type="route_manager" name="route_manager" output="screen"/>
```

Comment:
Let's not change existing interfaces , it's less disruptive and less surface area for testing. 

 I suggest keeping these as-is, so the route_manager can load a file to send new points or use the /map topic. So there's a "point provider".   Then use the rosparam config to toggle the mode. There is already a `mode` in the .yaml file, so just use this. And have a new separate launch file for using loading the random map config yaml.

Reply:
Restored the launch files to keep the existing interfaces same.. the dynamic route manager will be added to a new launch file